opendatahub-io/manifests
# Update Seldon to resulve issues with k8s 1.16/OCP 4.3

**Which issue is resolved by this Pull Request:**
Resolves #14 

@cliveseldon PTAL

**Description of your changes:**

Cherry picked Seldon update (https://github.com/kubeflow/manifests/commit/c4f51b1cc0b4854b6f6180d5a3bb021dd65691a8#diff-56fba03a2d55cd9b61df7e387d1d6d47) from manifests master and tested it on OCP 4.3

I also had to bump memory limit for seldon from 30Mi to 300Mi - it was getting OOM killed and add `cert-manager` to `kfctl_openshfit.yaml`. I also had to change the webhook port to `8443`

**Steps to verify:**

```
sed -i 's#uri: .*#uri: '$PWD'#' ./kfdef/kfctl_openshift.yaml
rm -rf kfdef/.cache kfdef/kustomize/ && kfctl build -V -f kfdef/kfctl_openshift.yaml 
kfctl apply -V -f kfdef/kfctl_openshift.yaml
```

You should see all pods in `istio-system`, `cert-manager` and `kubeflow` namespaces running without issue
